<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="datepad">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="datedisplay">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>6</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="australia">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mid_africa">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="north_africa">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="south_africa">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>400</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mozangbic">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>430</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="alaska">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="america">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>490</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="canada">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>520</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="south_america">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>550</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
